docs/_data/toc.yml

```diff
@@ -11,62 +11,61 @@
 
 - caption: shotgun
   children:
-  - page: /shotgun/
+  - page: shotgun
   - text: python-api
-    url: " https://developer.shotgunsoftware.com/python-api/"
+    url: "https://developer.shotgunsoftware.com/python-api/"
```

Comment:
please double check this - the white space you removed was there for a reason - it's related to the translations - see the comment at the top of the file :) 

If this whitespace isn't needed because of your changes, make sure to also update the comment at the top.

Reply:
Reverted these changes.